particles/PipeApps/source/RenderNotification.js

```diff
@@ -17,7 +17,15 @@ defineParticle(({UiParticle, html, log}) => {
       this.output({
         modality: 'notification',
         title: `Hello`,
-        text: `I'm a notification.`
+        text: `I'm a notification.`,
+        handler: `onNotificationClick`
+      });
+    }
+    onNotificationClick(event) {
```

Comment:
interestingly, the second notification always plays a sounds (and the renderer logs the log message), but it immediately disappear, unless i'm running with debugger.